Makefile

```diff
@@ -54,10 +70,10 @@ else
 endif
 
 quicktest:
-	$(PATH_WITH_HELM) go test -short ./ $(addprefix ./,$(GO_PACKAGES))
```

Comment:
This `${PATH_WITH_HELM}` variable is very important.  The tests will not pass without it set.